### DISCUSSION TITLE
AlDente 2.1.2 Working on Macbook Pro (Retina, 15-inch, Mid 2014) and Macbook Pro (16-inch, 2019)

### DISCUSSION THREAD

**Discussion by FFathil89** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#195] [General]
Hello,

I would like to inform that AlDente 2.1.2 is working fine on macOS Big Sur 11.2.3 in 15-inch Macbook Pro, Mid 2014. It is a base model with only integrated gpu.

For my experience, i need to enable the "Use the SMC Classic Key (Intel)" feature, quit and restart the macbook to allow the apps to properly work. If this feature is disabled, the macbook will not stop charging. I have also tested the same approach with my Macbook Pro (16-inch, 2019), and no issue so far.

Thank you for creating this app.

**Comment by MatthiasKerbl** [comment]
Hi @FFathil89,
I am glad you like AlDente. I have just updated our README with your model.

Matthias

**Comment by CatalysTim** [comment]
Hi @MatthiasKerbl,

I just wanted to add to and reiterate some specific details in @FFathil89's comment that weren't immediately obvious from the current README section.

I too am running Big Sur 11.2.3 on a **15-inch MacBook Pro, Mid-2014**, and despite several reboots and reinstallation attempts (per your suggestions in other threads) of **AlDente 2.2**, my MacBook didn't stop charging in both clamshell mode and non-clamshell mode. It was **necessary** to enable the "Use the SMC Classic Key (Intel)" option AND reboot the computer in order to stop the charging at the specified setpoint.

I'm not sure which other Intel-based MacBook models might also be exhibiting this behavior, but it might be a good idea to mention this explicitly in the README, so that other users might be able to avoid the trial and error that @FFathil89 and I needed to go through to figure this out. I only happened to stumble upon this thread by accident after going through every other thread here.

Thanks again for all the help on this excellent tool.

Best,
Tim

**Reply by MatthiasKerbl** [reply]
Hi @CatalysTim,

thanks for sharing your experience. After some testing, we found out that our new pathway to inhibit charging only works on 2016 MacBooks and newer. Therefore, I updated our README to clarify these differences.

Best regards,
Matthias
